README.md

```diff
@@ -171,10 +196,11 @@ BUMP_LEVEL=minor make release
 On certain versions of the tag hardware, the data from hydrophones is stored in raw format. Convert it before upload with the following script:
 ```console
 #!/bin/bash
-OUTDIR="../flac"
+OUTDIR="../flac/${PWD##*/}"
```

Comment:
This will create both files in the same folder. If subsequently you use `ceti s3upload {path}`, it will upload both raw files and .flac files. Not sure what's the best way to deal with this. The easiest is of course to `rm $f` after encoding. I guess its not too big of a deal, since this is just inside readme?

Reply:
Not sure I completely follow. There are no .raw files in the 'flac' folder that this script creates. If someone runs `ceti s3upload data/flac` only the .flac files will be uploaded.